src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,37 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
+import { actions } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useDispatch();
+  const { query, status } = useAppSelector(state => state.filter);
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
You can import every event type from react library like this `import { ChangeEvent } from 'react';`

It will make your code shorter and improve readability

```suggestion
  const handleQueryChange = (event: ChangeEvent<HTMLInputElement>) => {
```